File: Java/SneakyKnights/program03-testcases/TestCase01.java
// Sean Szumlanski
// COP 3503, Summer 2019

// ==============================
// SneakyKnights: TestCase01.java
// ==============================
// Tests that difficultyRating() is implemented correctly.


import java.io.*;
import java.util.*;

public class TestCase01
{
	public static void main(String [] args)
	{
		double difficulty = SneakyKnights.difficultyRating();
		System.out.println((difficulty < 1.0 || difficulty > 5.0) ? "fail whale :(" : "Hooray!");
	}
}

File: C/ListyFib/program03-Szum testcases/testcase01.c
// Sean Szumlanski
// COP 3502, Summer 2018

// ======================
// ListyFib: testcase01.c
// ======================
// This test case is a simple boundary check to make sure the difficultyRating()
// function is implemented correctly.


#include <stdio.h>
#include "ListyFib.h"

int main(void)
{
	if (difficultyRating() < 1.0 || difficultyRating() > 5.0)
		printf("fail whale :(\n");
	else
		printf("Hooray!\n");

	return 0;
}
